source/type/message.cpp

```diff
@@ -487,6 +510,17 @@ PtObj make_pt_obj(Type_e type, const std::string& uri, PT_Data& pt_data) {
     }
 }
 
+void Impact::sort_application_patterns() {
+    boost::sort(
+        this->application_patterns,
+        [](const nt::disruption::ApplicationPattern& ap1, const nt::disruption::ApplicationPattern& ap2) -> bool {
+            return ap1.application_period.begin() < ap2.application_period.begin();
```

Comment:
Do you need a specific order on application patterns with the `same application_period.begin()` ?

Reply:
I prepare it for navitia interface.
for optimization reasons, I do it here instead of doing it in "pb_converter"